mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-parties-7.2.0.plantuml

```diff
@@ -74,149 +75,133 @@ end box
 
-        activate ALS_API
-        ALS_API -> ALS_API: Validate request against Mojaloop Interface Specification. \n<color #FF0000><b>Error code:</b> 300x, 310x</color>
+        ALS_API -> ALS_API: Validate request against\nMojaloop Interface Specification\n<color #red><b>Error code:</b> 300x, 310x</color>
         ALS_API -> ALS_PARTICIPANT_HANDLER: Process Participant Callback Information for PUT
-        deactivate ALS_API
 
         '********************* Request Parties Information - END ************************
 
         '********************* Get PayerFSP Participant Information - START ************************
 
-        ALS_PARTICIPANT_HANDLER -> ALS_CENTRALSERVICE_ENDPOINT_CONFIG_DAO: Retrieve the PayerFSP Participant Callback End-point. \n<color #FF0000><b>Error code:</b> 200x</color>
+        ALS_PARTICIPANT_HANDLER -> ALS_CENTRALSERVICE_ENDPOINT_CONFIG_DAO: Retrieve the PayerFSP Participant Callback Endpoint\n<color #red><b>Error code:</b> 200x</color>
         activate ALS_CENTRALSERVICE_ENDPOINT_CONFIG_DAO
-        ALS_CENTRALSERVICE_ENDPOINT_CONFIG_DAO -> CENTRALSERVICE_API: Retrieve the PayerFSP Participant Callback End-point - GET /participants/{FSPIOP-Source}/endpoints. \n<color #FF0000><b>Error code:</b> 200x, 310x, 320x</color>
+        ALS_CENTRALSERVICE_ENDPOINT_CONFIG_DAO -> CENTRALSERVICE_API: Retrieve the PayerFSP Participant Callback Endpoint\nGET - /participants/{FSPIOP-Source}/endpoints\n<color #red><b>Error code:</b> 200x, 310x, 320x</color>
         activate CENTRALSERVICE_API
-        CENTRALSERVICE_API -> ALS_CENTRALSERVICE_ENDPOINT_CONFIG_DAO: List of PayerFSP Participant Callback End-points
+        CENTRALSERVICE_API --> ALS_CENTRALSERVICE_ENDPOINT_CONFIG_DAO: List of PayerFSP Participant Callback Endpoints
         deactivate CENTRALSERVICE_API
-        ALS_CENTRALSERVICE_ENDPOINT_CONFIG_DAO -> ALS_PARTICIPANT_HANDLER: List of PayerFSP Participant Callback End-points
+        ALS_CENTRALSERVICE_ENDPOINT_CONFIG_DAO --> ALS_PARTICIPANT_HANDLER: List of PayerFSP Participant Callback Endpoints
         deactivate ALS_CENTRALSERVICE_ENDPOINT_CONFIG_DAO
 
-        ALS_PARTICIPANT_HANDLER -> ALS_PARTICIPANT_HANDLER: Match PayerFSP Participant Callback End-points for `FSPIOP_CALLBACK_URL_PARTIES_PUT`
+        ALS_PARTICIPANT_HANDLER -> ALS_PARTICIPANT_HANDLER: Match PayerFSP Participant Callback Endpoints for\nFSPIOP_CALLBACK_URL_PARTIES_PUT
 
         '********************* Get PayerFSP Participant Information - END ************************
 
-        ALS_PARTICIPANT_HANDLER -> ALS_API: Send Participant Information to PayerFSP.
-        activate ALS_API
-        ALS_API --> PAYER_FSP: Callback with Parties Information: PUT - /parties/{TYPE}/{ID}?currency={CURRENCY}
-        deactivate ALS_API
+        ALS_PARTICIPANT_HANDLER --> ALS_API: Return Participant Information to PayerFSP
+        ALS_API ->> PAYER_FSP: Callback with Parties Information:\nPUT - /parties/{TYPE}/{ID}?currency={CURRENCY}
```

Comment:
We use '-->' to denote an async callback response in our other sequence diagrams do we not?